S7.Net/PLC.cs

```diff
@@ -133,10 +145,18 @@ public void Close()
             }
         }
 
-        private void ConfigureNetworkStream(NetworkStream networkStream)
+        private void ConfigureConnection()
+        {
+            if (tcpClient != null)
+            {
+                ConfigureConnection(tcpClient);
+            }
+        }
+
+        private void ConfigureConnection(TcpClient client)
```

Comment:
Either make this static or let it work on the `tcpClient` member instead of passing the `TcpClient client` argument.